redaxo/src/addons/media_manager/pages/effects.php

```diff
@@ -17,14 +17,6 @@
 $info = '';
 $warning = '';
 
-//-------------- delete cache on effect changes or deletion
-if ((rex_request('func') != '' || $func == 'delete')
-     && $type_id > 0
-) {
-    $counter = rex_media_manager::deleteCacheByType($type_id);
```

Comment:
ist diese counter variable ungenutzt?

Reply:
Ja, wurde nur mal in der auskommentierten Zeile direkt hier drunter verwendet.